packaging_automation/update_docker.py

```diff
@@ -0,0 +1,147 @@
+import argparse
+import uuid
+import os
+from datetime import datetime
+
+from github import Github
+
+from .common_tool_methods import (process_template_file, write_to_file, run, initialize_env)
+
+REPO_OWNER = "citusdata"
+PROJECT_NAME = "docker"
+
+
+def update_docker_file_for_latest_postgres(project_version: str, template_path: str, exec_path: str,
+                                           postgres_version: str):
+    content = process_template_file(project_version, template_path,
+                                    "latest/latest.tmpl.dockerfile", postgres_version)
```

Comment:
1) I think we can put all `tmpl` file paths into a file like `tmpl_file_paths.py` and read them from there when needed (update_docker.py & `test.py` files that needs to access).

2) We can put `dest_file_name` suffixes (like `/docker-compose.yml` &  `/Dockerfile`) into some global variables in this file  (`update_docker.py`).